<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 20, 20);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/* Set background colors and properties */
QFrame#mainFrame{
background-color: #f0f0f0;
}




QFrame#contentFrame{
background-color: #fff;
}

QFrame#leftMenuFrame{
background-color: #36647b;
}

QWidget#page_License {
background-color: #f0f0f0;
}

QPushButton:checked{
color:rgb(255, 166, 0);
background-color: rgb(88, 88, 88);
}
QCheckBox{
font:  &quot;Poppins&quot;;
font-weight: bold;
font-size: 8pt;
color: #D7DBEC;
}

/* Global styles for buttons */
QPushButton{
/*background-color: #383838;*/
border-radius: 5px;
border:2px solid #D7DBEC;;
font-weight: bold;
font-size: 8pt;
background:#2f373d; 
/*font-family: Noto Sans;*/
font: 75 10pt &quot;Poppins&quot;;
color: #fff;
height:40px;
width:150px;
}

QPushButton:hover{
color:rgb(255, 166, 0);
}

/* Styles for Tab Widgets */
QTabWidget::pane {
border:1px solid lightgray;
/*background: rgb(245, 245, 245);*/

} 

QTabBar::tab {
/*background: rgb(230, 230, 230);*/ 
background-color: #191C24;
color:#F1F1F2;
border: 1px solid lightgray; 
padding: 10px 30px 10px 30px;
margin:1px;
} 
QTabBar::tab:hover {
	background-color:#343434;
	border-top-left-radius: 5px; 
    border-top-right-radius: 5px; 
    border-bottom-left-radius: 0px; 
    border-bottom-right-radius: 0px; 
} 

QTabBar::tab:selected {
	background: rgb(88,88,88);
	color:rgb(255, 166, 0);
	margin-bottom: -1px; 
}

/* QLabel settings */
QLabel#SettingBg1{
border-radius: 5px;
border-style: solid;
border-width: 2px;
border-color: rgba(40, 40, 40, 0.2);
}

QLabel#SettingBg2{
border-radius: 5px;
border-style: solid;
border-width: 2px;
border-color: rgba(40, 40, 40, 0.2);
}

QLabel#SettingBg3{
border-radius: 5px;
border-style: solid;
border-width: 2px;
border-color: rgba(40, 40, 40, 0.2);
}

QLabel#SettingBg4{
font: 14pt &quot;Poppins&quot;;
border-radius: 5px;
border-style: solid;
border-width: 2px;
border-color:  rgba(40, 40, 40, 0.2);
}

QLabel {
font-size: 8pt;
font:75  &quot;Poppins&quot;;
font-weight: bold;
color: #D7DBEC;
/*font-family: Noto Sans;*/
}

/* Global combobox settings */
QComboBox{
text-align: center;
border-radius: 5px;
border:2px solid #D7DBEC;
color: #D7DBEC;
padding: 2px;
font-weight: bold;
font-size: 8pt;
height:40px;
width:150px;
}

QComboBox::drop-down {
border: 0px; /* This seems to replace the whole arrow of the combo box */

}

QComboBox QAbstractItemView {
	text-align: center;
	qproperty-alignment: AlignCenter;
    background-color: white; /* Background color */
    color: #F1F1F2; /* Text color */
}

QComboBox::down-arrow {
image: url(img/Arrows-Down-4-icon.png);
width: 10px;
height: 10px;
}

QComboBox:hover{
color:rgb(255, 166, 0);
background-color:white;
/*border: 2px solid #252429;
color: #252429;*/
}

QTableView{
color: rgb(31, 31, 31);
background-color: rgb(209, 209, 209);
}

QHeaderView::section { 
font: 75 10pt &quot;Poppins&quot;;
color: white;
background-color:#989898; 
}

QDateTimeEdit{
background-color: rgb(81, 81, 81);
color: rgb(255, 255, 255);
}

QTableView::item { 
font: 75 12pt &quot;Poppins&quot;;
color:white; 
background:#2f373d; 
font-weight:900; 
}
QTableView::item:selected {
font: 75 12pt &quot;Poppins&quot;; 
color:black; 
background:#7cc8b9; 
font-weight:900; 
}
QTableView{
color: rgb(31, 31, 31);
background-color: rgb(25, 28, 36);
}

QTextEdit{
background-color: rgb(25, 28, 36);
color: rgb(255, 255, 255);
font: 8pt &quot;Poppins&quot;;
}</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>951</width>
      <height>841</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>951</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color: rgb(25, 28, 36);
}
QPushButton{
 font: 12pt &quot;Poppins&quot;;
 border:none;
}
QPushButton:checked{
color: rgb(248, 172, 35);
background-color: rgb(0, 0, 0);
border-color: rgb(248, 172, 35);
border-color: rgb(251, 220, 175);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>24</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>PLC Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>90</y>
       <width>191</width>
       <height>751</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
 font: 12pt &quot;Poppins&quot;;
 color:white;
 text-align:left;
 height:40px;
 border:none;
 padding-left:10px;
}
QPushButton:checked{
color:rgb(255, 166, 0);
background-color: rgb(88, 88, 88);

}

QPushButton:hover{
background-color:#343434;
}

QWidget{
background-color: #191C24;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="navHome">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
          <property name="icon">
           <iconset resource="Icons/main_icons.qrc">
            <normaloff>:/Icons/home.png</normaloff>:/Icons/home.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="navExport">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Export</string>
          </property>
          <property name="icon">
           <iconset resource="Icons/main_icons.qrc">
            <normaloff>:/Icons/export.png</normaloff>:/Icons/export.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="navLog">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Log</string>
          </property>
          <property name="icon">
           <iconset resource="Icons/main_icons.qrc">
            <normaloff>:/Icons/lod.png</normaloff>:/Icons/lod.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="navHelp">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Help</string>
          </property>
          <property name="icon">
           <iconset resource="Icons/main_icons.qrc">
            <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="navAbout">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>About</string>
          </property>
          <property name="icon">
           <iconset resource="Icons/main_icons.qrc">
            <normaloff>:/Icons/lod.png</normaloff>:/Icons/lod.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="navImp">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Import Excel</string>
          </property>
          <property name="icon">
           <iconset resource="Icons/main_icons.qrc">
            <normaloff>:/Icons/import.png</normaloff>:/Icons/import.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>118</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>741</width>
       <height>751</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="homePage">
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>20</y>
         <width>751</width>
         <height>741</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
background-color: #191C24;
}
</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>631</width>
          <height>123</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnDisconnect">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextEdit" name="textStatus">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>180</y>
          <width>631</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="exportPage">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>To :</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="from_time">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="minimumDate">
        <date>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="to_time">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
       <property name="date">
        <date>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
      <widget class="QPushButton" name="export_btn">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>90</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>10</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Export Excel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="show_data_btn">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>160</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>10</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Show Data</string>
       </property>
      </widget>
      <widget class="QTableView" name="table_view">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>541</width>
         <height>511</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>From :</string>
       </property>
      </widget>
      <zorder>label_3</zorder>
      <zorder>to_time</zorder>
      <zorder>export_btn</zorder>
      <zorder>show_data_btn</zorder>
      <zorder>table_view</zorder>
      <zorder>label_2</zorder>
      <zorder>from_time</zorder>
     </widget>
     <widget class="QWidget" name="logPage">
      <widget class="QTextEdit" name="logField">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>731</width>
         <height>681</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnClearLog">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="helpPage">
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>721</width>
         <height>721</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
background-color: #191C24;
}</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="aboutPage">
      <widget class="QWidget" name="widget_7" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>721</width>
         <height>721</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
background-color: #191C24;
}</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="importPage">
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>-20</y>
         <width>741</width>
         <height>621</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
background-color: #191C24;
}
</string>
       </property>
       <widget class="QPushButton" name="btnImpExcel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>210</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>10</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </widget>
      <widget class="QTextEdit" name="logImp">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>650</y>
         <width>741</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Icons/main_icons.qrc"/>
 </resources>
 <connections/>
</ui>
